app/controllers/users_controller.rb

```diff
@@ -3,4 +3,10 @@ class UsersController < ApplicationController
 
   def show
   end
+
+  private
+
+  def expose_user_id_param!
```

Comment:
What do you think about creating users/pages_controller.rb with index or show method instead of this controller to avoid overriding the parameters?